### PR TITLE
Feature/survey-feedback card

### PR DESCRIPTION
**Pull request by justinkook**
New Temporary Survey Card that links to: https://bitpayux.typeform.com/to/z81Jp7

Replaces the smiley face card for now until survey testing period is finished.

### CODE REVIEW

File: src/pages/home/home.html [hunk 1]
```diff
@@ -99,7 +99,8 @@
           Access denied
         </div>
         <div class="server-message-body" translate>
-          Unfortunately, your country/region has banned the use or acceptance of cryptocurrencies as a valid form of payment and we are therefore unable to service you.
```
Comment (gabrielbazan7) [inline src/pages/home/home.html]: keep this string in a single line so we don't have a new translation

File: src/pages/home/home.html [hunk 2]
```diff
@@ -157,7 +158,9 @@
           <label-tip type="warn" *ngIf="wallet.isComplete() && wallet.needsBackup">
             <span label-tip-title translate>Wallet Needs Backup</span>
             <div label-tip-body>
-              <span translate>Before retrieving your wallet address, it's recommended that you first write down your recovery phrase and store it securely so that your wallet can be recovered in the case your device was lost or stolen.</span>
```
Comment (gabrielbazan7) [inline src/pages/home/home.html]: Also this one

File: src/pages/includes/survey-feedback/survey-feedback.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<action-card *ngIf="isShowRateCard">
+  <div action-card-header item-end class="icon-right">
+    <button ion-button clear icon-only item-end color="grey" (click)="hideCard()">
+      <ion-icon name="close"></ion-icon>
+    </button>
+  </div>
+  <div action-card-title translate>Got a minute?</div>
+  <div action-card-body translate>{{feedbackCardBody}}</div>
```
Comment (gabrielbazan7) [inline src/pages/includes/survey-feedback/survey-feedback.html]: remove translate

File: src/pages/includes/survey-feedback/survey-feedback.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  <div action-card-button translate (click)="openSurveyInBrowser()" tappable>
```
Comment (gabrielbazan7) [inline src/pages/includes/survey-feedback/survey-feedback.html]: same here

File: src/pages/includes/survey-feedback/survey-feedback.html [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+    <span class="survey-button">Take Survey</span>
```
Comment (gabrielbazan7) [inline src/pages/includes/survey-feedback/survey-feedback.html]: this one needs translate

Comment (justinkook) [reply src/pages/includes/survey-feedback/survey-feedback.html]: done

File: src/pages/includes/survey-feedback/survey-feedback.html (hunk 4 above)
Comment (justinkook) [reply src/pages/includes/survey-feedback/survey-feedback.html]: done

File: src/pages/includes/survey-feedback/survey-feedback.html (hunk 3 above)
Comment (justinkook) [reply src/pages/includes/survey-feedback/survey-feedback.html]: done

File: src/pages/home/home.html (hunk 2 above)
Comment (justinkook) [reply src/pages/home/home.html]: done

File: src/pages/home/home.html (hunk 1 above)
Comment (justinkook) [reply src/pages/home/home.html]: done, was auto html formatter from vscode

**Pull request closed without merging**